bip-0039/portuguese_brazilian.txt

```diff
@@ -0,0 +1,2048 @@
+abacaxi
+abaixo
+abdome
+abelha
+abismo
+abono
+abrigo
+absinto
+absurdo
+abutre
+acaso
+acelga
+acento

```

Comment:
I suggest changing words like "abdome" which can be written in more than one form, and words like "acento", which sounds identical to "assento".